Flaky unit test in app/api/entities/specs/entites.spec.js

A specific unit test sometimes fails on the github CI. Usually, in those cases one cannot reproduce the error locally, and re-running the unit tests resolves the issue without any change.

An example error report from the CI:
```
Summary of all failing tests
FAIL app/api/entities/specs/entities.spec.js (26.183s)
  ● entities › save › when generatedToc is undefined › should not replicate the value to all languages

    TypeError: Cannot read property 'template' of undefined

      134 |   const docLanguages = await this.getAllLanguages(entity.sharedId);
      135 |   if (
    > 136 |     docLanguages[0].template &&
          |                     ^
      137 |     entity.template &&
      138 |     docLanguages[0].template.toString() !== entity.template.toString()
      139 |   ) {

      at Object.template [as updateEntity] (api/entities/entities.js:136:21)
          at runMicrotasks (<anonymous>)
      at Object.save (api/entities/entities.js:416:7)


Test Suites: 1 failed, 603 passed, 604 total
Tests:       1 failed, 3793 passed, 3794 total
Snapshots:   423 passed, 423 total
Time:        523.648s
Ran all test suites in 2 projects.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/app/api/entities/specs/entities.spec.js
+++ b/app/api/entities/specs/entities.spec.js
@@ -284,7 +284,7 @@ describe('entities', () => {
       });
     });
 
-    it('should ignore the permissions parameter', () => {
+    it('should ignore the permissions parameter', async () => {
       const doc = {
         _id: unpublishedDocId,
         sharedId: 'other',
@@ -292,12 +292,12 @@ describe('entities', () => {
         permissions: [],
       };
 
-      entities.save(doc, { language: 'en' }).then(updatedDoc => {
-        expect(updatedDoc.permissions).toEqual([
-          expect.objectContaining({ refId: 'user1' }),
-          expect.objectContaining({ refId: 'user2' }),
-        ]);
-      });
+      const updatedDoc = await entities.save(doc, { language: 'en' });
+
+      expect(updatedDoc.permissions).toEqual([
+        expect.objectContaining({ refId: 'user1' }),
+        expect.objectContaining({ refId: 'user2' }),
+      ]);
     });
 
     describe('when generatedToc is undefined', () => {

